xml_converter/intigration_tests/inputs/xml_animation_speed/xml_file.xml

```diff
@@ -0,0 +1,13 @@
+<OverlayData>
+    <MarkerCategory DisplayName="My Category" Name="mycategory">
+    </MarkerCategory>
+
+    <POIs>
+        <Trail AnimSpeed="0" Type="mycategory" />
+        <Trail AnimSpeed="3.14" Type="mycategory" />
+        <Trail AnimSpeed="123.456" Type="mycategory" />
+        <Trail AnimationSpeed="0" Type="mycategory" />
+        <Trail AnimationSpeed="3.14" Type="mycategory" />
+        <Trail AnimationSpeed="123.456" Type="mycategory" />
```

Comment:
Can this be negative? With the animation running reverse?

Reply:
I don't believe it can. The BlishHud docs do not specify how they implement it but in Godot, there is a property of the animation player called `playback_speed` which also has a default of 1.0 that is a speed scaling ratio. That is likely the field that would be edited.  This also does not say in the documentation if it can be negative.

Reply:
https://docs.godotengine.org/en/3.3/classes/class_animationplayer.html#class-animationplayer-method-play